test-case/src/ContaoTestCase.php

```diff
@@ -63,6 +63,14 @@ public static function tearDownAfterClass(): void
         unset(self::$tempDirs[$key]);
     }
 
+    protected function tearDown(): void
+    {
+        // Unset TL_CONFIG, since we populate it in loadDefaultConfiguration
+        unset($GLOBALS['TL_CONFIG']);
```

Comment:
Maybe you can also delete the `tearDown()` methods in the tests that do nothing more than this?

Reply:
Yes, will do once this is merged and then merged upstream.

Reply:
Please rebase this PR to the 4.13 branch and make the changes here.

Reply:
Done in 6cf9fe27b7224529121dbb6ba0c42b3c6c6cb964